fiboa_cli/datasets/fieldscapes_kenya_2022.py

```diff
@@ -0,0 +1,138 @@
+# TEMPLATE FOR A FIBOA CONVERTER
+#
+# Copy this file and rename it to something sensible.
+# The name of the file will be the name of the converter in the cli.
+# If you name it 'de_abc' you'll be able to run `fiboa convert de_abc` in the cli.
+
+from ..convert_utils import convert as convert_
+
+# File to read the data from
+# Can read any tabular data format that GeoPandas can read through read_file()
+# Supported protcols: HTTP(S), GCS, S3, or the local file system
+
+# Local URI added to the repository for initial conversion, Original Source https://beta.source.coop/esa/fusion-competition/
+URI = "/home/byteboogie/work/labwork_hkerner/fieldscapes/kenya/boundaries_kenya_2022.gpkg"
+
+# Unique identifier for the collection
+ID = "fieldscapes_kenya_2022"
+# Title of the collection
+TITLE = "Field boundaries for Kenya (Fieldscapes)"
+# Description of the collection. Can be multiline and include CommonMark.
+DESCRIPTION = """ The dataset contains field boundaries for the Kenya."""
+# Bounding box of the data in WGS84 coordinates
+BBOX = [34.2902402, -3.73181826832162, 38.721089102432, 1.21940779553077]
+
+# Provider name, can be None if not applicable, must be provided if PROVIDER_URL is provided
+PROVIDER_NAME = "ECASS Project"
+# URL to the homepage of the data or the provider, can be None if not applicable
+PROVIDER_URL = "https://ecass-project-documentation.readthedocs.io/en/latest/modules/data_access.html#data-access"
+# Attribution, can be None if not applicable
+ATTRIBUTION = "Copyright 2022, PULA"
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+LICENSE = "GPL-2.0-or-later"
+# 2. a STAC Link Object with relation type "license"
+# LICENSE = {"title": "CC-BY-4.0", "href": "https://creativecommons.org/licenses/by/4.0/", "type": "text/html", "rel": "license"}
+
+# Map original column names to fiboa property names
+# You also need to list any column that you may have added in the MIGRATION function (see below).
+COLUMNS = {
+    "Case_ID": "id",
+    "Crop_Type": "crop_name",
+    "geometry": "geometry"
+}
+
+# Add columns with constant values.
+# The key is the column name, the value is a constant value that's used for all rows.
+ADD_COLUMNS = {
+    "determination_datetime": "2021-01-01T00:00:00Z"
```

Comment:
2022 and 2021 determination doesn't quite fit together.

Reply:
Right this is slight mistake.